boards/b-l072z-lrwan1/dist/openocd.cfg

```diff
@@ -1,3 +1,3 @@
 source [find target/stm32l0.cfg]
 
-reset_config trst_and_srst srst_nogate connect_assert_srst
+reset_config srst_only connect_assert_srst
```

Comment:
Change not related.

Reply:
Ok, I dropped the commit, it was the same change as #8790, so can be reviewed and merged from there.